.env

```
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# use OVOS alpha channel
+OVOS_CHANNEL="alpha"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)